docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,326 @@
+.. include:: ../../common_links.inc
+
+.. _ugrid model:
+
+The Mesh Data Model
+*******************
+
+.. important::
+
+        This page is intended to summarise the essentials that Iris users need
+        to know about meshes. For exhaustive details on UGRID itself:
+        `visit the official UGRID conventions site`__.
+
+Evolution, not revolution
+=========================
+Mesh support has been designed wherever possible to fit within the existing
+Iris model. Meshes concern only the spatial location of data, and can
+optionally be limited to just the horizontal location (e.g. X and Y). Other
+dimensions such as time or ensemble member (and often vertical levels)
+retain their familiar structured format.
+
+The UGRID conventions themselves are designed as an addition to the existing CF
+conventions, which are at the core of Iris' philosophy.
+
+What's Different?
+=================
+
+The mesh format represents data's geographic location using an **unstructured
+mesh**. This has significant pros and cons when compared to a structured grid.
+
+.. contents::
+   :local:
+
+The Detail
+----------
+..
+    The diagram images are SVG's, so editable by any graphical software
+     (e.g. Inkscape). They were originally made in MS PowerPoint.
+
+    Uses the IBM Colour Blind Palette (see
+     http://ibm-design-language.eu-de.mybluemix.net/design/language/resources/color-library
+     )
+
+Structured Grids
+~~~~~~~~~~~~~~~~
+Assigning data to locations using a structured grid is essentially an act of
+matching coordinate arrays to each dimension of the data array. The data can
+also be represented as an area (instead of a point) by including a bounds array
+for each coordinate array.
+
+.. figure:: images/data_structured_grid.svg
+   :alt: Diagram of how data is represented on a structured grid
+
+   Data on a structured grid
+
+   :download:`full size image <images/data_structured_grid.svg>`
+
+Unstructured Meshes
+~~~~~~~~~~~~~~~~~~~
+The most basic element in a mesh is the 0-dimensional **node**: a single
+location in space. Every node in the mesh is defined by indexing the
+1-dimensional X and Y (and optionally Z) coordinate arrays (the
+``node_coordinates``) - e.g. ``(x[3], y[3])`` gives the position of the fourth
+node. Note that this means each node has its own coordinates, independent of
+every other node. Since nodes can be anywhere in the space - **unstructured** -
+the position in the array has nothing to do with spatial position.
+
+If data is assigned to node location it must be stored in a 1D array of equal
+length to the number of nodes in the mesh. ``data[3]`` is at the position:
+``(x[3], y[3])``.
```

Comment:
This sentence starting with "data[3]" seemed to me to appear somewhat 'out of the blue', as a bit of a shock.
Possibly all it needs is a "Thus, for example... " before it ?

Reply:
a0b405c3b4432e6e172616620cb6a1563b083a57